From e91c8a98ae127f4ed2c50df3913c9af8950359ce Mon Sep 17 00:00:00 2001
From: Emilian Bold <emi@apache.org>
Date: Fri, 9 Nov 2018 13:44:46 +0200
Subject: [PATCH 19/32] Excludes release-toplevel folder (which has README.html
 and CREDITS) from the zip
License: Copyright (c) 2018, 2019 Emilian Marius Bold
 
 Permission to use, copy, modify, and distribute this software for any
 purpose with or without fee is hereby granted, provided that the above
 copyright notice and this permission notice appear in all copies.
 
 THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
 WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
 MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
 ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.

---
 nb/ide.branding/build.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/nb/ide.branding/build.xml b/nb/ide.branding/build.xml
index 223ce17800..b5a6775fd2 100644
--- a/nb/ide.branding/build.xml
+++ b/nb/ide.branding/build.xml
@@ -31,9 +31,11 @@ Contributor(s): Vincent Brabant, Maxym Mykhalchuk
 
   <target name="netbeans-extra">
 
+<!-- CoolBeans doesn't need these
     <copy todir="${netbeans.dest.dir}">
       <fileset dir="release-toplevel"/>
     </copy>
+-->
     
     <locjar warnMissingDir="true"
       basedir="core.startup/src"
-- 
2.21.0

